### DISCUSSION TITLE
Inconsistent border in stack activity

### DISCUSSION THREAD

**Discussion by quyleanh** [community/community#22213] [Accessibility]
<p>I think the upper highlighted one should be the same with the lower highlighted one.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/2/626124d040648f5149a4484f0c0771fcb69266f2.png" data-download-href="/uploads/short-url/e2iV9aLGwxMc8AUCUG0J0IRkx34.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/9365035/181069561-bf0d33c4-6f3f-42a9-9717-0c561e7886a7.png" alt="image" data-base62-sha1="e2iV9aLGwxMc8AUCUG0J0IRkx34" width="616" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/2/626124d040648f5149a4484f0c0771fcb69266f2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">662×537 38.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by gaknoll** [comment, marked as answer]
<p>We appreciate the attention to detail and for taking the time to share this with us <a class="mention" href="/quyleanh">@quyleanh</a>! Can I ask you what page this is on? And are you by chance using any browser extensions such as <code>refined-github</code>?</p>

**Comment by quyleanh** [comment]
<p>Thank you for your kind response.<br>
Yes, actually I’m using <code>refined-github</code>.<br>
When I turn it off, my Github homepage is fine. I’m sorry for trouble you.</p>
<p>This thread should be closed and marked as Solve.</p>

**Comment by gaknoll** [comment]
<p>Glad that resolved it for you <a class="mention" href="/quyleanh">@quyleanh</a>! And not a trouble at all. We appreciate you taking the time to share issues like this should they in fact be a bug on our side :pray:</p>

**Comment by cehfisher** [comment]
Hi @queenofcorgis - this stale issue is resolved [per this comment by the original poster](https://github.com/orgs/community/discussions/22213#discussioncomment-3235930). Could you please close this issue?
